src/sentry/static/sentry/app/views/settings/organizationTeams/index.tsx

```diff
@@ -0,0 +1,240 @@
+import {RouteComponentProps} from 'react-router/lib/Router';
+import React from 'react';
+import * as Sentry from '@sentry/browser';
+import debounce from 'lodash/debounce';
+
+import {Client} from 'app/api';
+import {DEFAULT_DEBOUNCE_DURATION} from 'app/constants';
+import {Organization, Team} from 'app/types';
+import {addErrorMessage} from 'app/actionCreators/indicator';
+import {removeAtArrayIndex} from 'app/utils/removeAtArrayIndex';
+import {replaceAtArrayIndex} from 'app/utils/replaceAtArrayIndex';
+import {t} from 'app/locale';
+import parseLinkHeader from 'app/utils/parseLinkHeader';
+import withOrganization from 'app/utils/withOrganization';
+
+import OrganizationTeams from './organizationTeams';
+
+const TEAM_PAGE_LIMIT = 25;
+
+type Props = {
+  organization: Organization;
+} & RouteComponentProps<{orgId: string}, {}>;
+
+type State = {
+  activeTeams: Team[] | null;
+  otherTeams: Team[] | null;
+  hasMoreTeams: boolean;
+};
+
+class OrganizationTeamsContainer extends React.Component<Props, State> {
+  state: State = {
+    activeTeams: null,
+    otherTeams: null,
+    hasMoreTeams: false,
+  };
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  componentWillUnmount() {
+    this.allApi.clear();
+    this.activeTeamsApi.clear();
+  }
+
+  allApi = new Client();
+  activeTeamsApi = new Client();
+
+  async fetchData() {
+    this.setState({
+      activeTeams: null,
+      otherTeams: null,
+    });
+
+    this.fetchAllTeams();
+    this.fetchActiveTeams();
+  }
+
+  async fetchActiveTeams() {
+    const {orgId} = this.props.params;
+
+    this.activeTeamsApi.clear();
+
+    try {
+      const resp = await this.activeTeamsApi.requestPromise(
+        `/organizations/${orgId}/user-teams/`
+      );
```

Comment:
Do we need to worry about pagination with the active teams as well?

Reply:
The endpoint is not paginated - I looked at discover, and the endpoint generally responds fairly quickly, the only case where it doesn't is for superusers iirc.